lib/os/fdtable.c

```diff
@@ -53,7 +53,9 @@ static struct fd_entry fdtable[CONFIG_POSIX_MAX_FDS] = {
 		.refcount = ATOMIC_INIT(1)
 	},
 #else
+	{
 	0
+	},
```

Comment:
should indent properly.

```suggestion
	{
		0
	},
```